app/controllers/concerns/jobseekers/wizardable.rb

```diff
@@ -55,4 +62,15 @@ def declarations_params(params)
                       .require(:jobseekers_job_application_declarations_form)
                       .permit(:banned_or_disqualified, :close_relationships, :close_relationships_details, :right_to_work_in_uk)
   end
+
+  def reference_details
```

Comment:
Let's not use the word 'details' unless it refers to a JobApplicationDetail object - I'll get confused otherwise. Maybe it could be called reference_fields or reference_info or reference_labels.